src/App.jsx

```diff
@@ -5,15 +5,29 @@ import AddBlogTitle from './AddBlogTitle.jsx'
 import Search from './Search.jsx'
 
 function App() {
-    const [newTitle, setNewTitle] = react.useState();
+    const initialTitles = ["Lesson 1.1 Project setup",
+        "Lesson 1.2 React DOM and components",
+        "Lesson 1.3 Form input, props, state and callbacks",
+        "Lesson 1.4 Lifting state, controlled components, props handling",
+        "Lesson 1.5 Side effects, custom hooks, fragments",
+    ]
+    const blogTitlesFromLocalStorage = JSON.parse(localStorage.getItem('savedBlogTitles')) || initialTitles
```

Comment:
Having the logical or operator here is important for a new visitor to the site or one that just cleared their storage. This way if `JSON.parse(localStorage.getItem('savedBlogTitles'))` returns null then javascript will use the `initialTitles` value instead.